util/unifyfs/src/unifyfs-rm.c

```diff
@@ -539,6 +657,45 @@ static size_t construct_server_argv(unifyfs_args_t* args,
     return argc;
 }
 
+/**
+ * @brief Default data stage launch routine
+ *
+ * @param args        The command-line options
+ * @param stage_args  unifyfs-stage argument vector to be filled
```

Comment:
rename "vector" to "array"

Reply:
You should also document the minimum size `stage_args[]` should be (6 entries?)

Reply:
RE vector/array: That's the way the description for the existing function construct_server_argv was written.  Also I think vector is at least as correct as array in this instance.  

Let's update the documentation of all the functions together as a separate issue/branch/PR (unless construct_server_args is documented somewhere that I missed and that construct_stage_argv doesn't match that).